ptvsd/wrapper.py

```diff
@@ -396,7 +405,10 @@ def make_packet(self, cmd_id, args):
         with self.lock:
             seq = self.seq
             self.seq += 1
-        s = '{}\t{}\t{}\n'.format(cmd_id, seq, args)
+        if is_instance_of_unicode(cmd_id) or is_instance_of_unicode(args):
+            s = u'{}\t{}\t{}\n'.format(cmd_id, seq, args)
```

Comment:
The `u''` string literal prefix is available in Python 3 (since 3.4) as well, so you should be able combine the two branches, no?